rmf_traffic/include/rmf_traffic/schedule/StubbornNegotiator.hpp

```diff
@@ -56,6 +56,27 @@ class StubbornNegotiator : public Negotiator
   ///   The Participant who wants to be stubborn.
   StubbornNegotiator(std::shared_ptr<const Participant> participant);
 
+  using UpdateVersion = rmf_utils::optional<ItineraryVersion>;
+
+  /// Add a set of acceptable wait times.
+  ///
+  /// \param[in] wait_times
+  ///   A list of the wait times that would be accepted for negotiation
+  ///
+  /// \param[in] approval_cb
+  ///   A callback that will be triggered
```

Comment:
Nit: would it be helpful to provide more info on the expectations of this callback?

Reply:
Whoops, I trailed off mid-sentence. That was definitely meant to say more. Finished here: c0090c6

Reply:
Also unfortunately GitHub is demanding a follow-up approval now that I've made a commit.